akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -148,6 +207,21 @@ final class ORSet[A] private[akka] (
 
   type T = ORSet[A]
 
+  // Optimization for add/remove followed by merge, which is very common
+  // and merge should just fast forward to the new set
+  private var ancestor: ORSet[A] = null
```

Comment:
So it is guaranteed that only one thread will access that field at any time, if I understood this correctly? I would still be more happy if that would be commented :)